Orange/widgets/visualize/owheatmap.py

```diff
@@ -174,24 +174,35 @@ def color_palette_model(palettes, iconsize=QSize(64, 16)):
 
 
 def color_palette_table(colors, samples=255,
-                        threshold_low=0.0, threshold_high=1.0, gamma=None):
+                        threshold_low=0.0, threshold_high=1.0,
+                        underflow=None, overflow=None,
+                        gamma=None):
```

Comment:
Isn't there something like this, but apparently in much poorer condition, in `Orange.widget.utils.colorpalette.ExtendedContinuousPaletteGenerator`? Does it do (approximately) the same thing?